utils/auto_tls.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import (
+	"crypto/tls"
+	"golang.org/x/crypto/acme/autocert"
+)
+
+func AutoTLS(dirCache string, hosts ...string) tls.Config {
```

Comment:
warning: exported function AutoTLS should have comment or be unexported (golint)